app/src/main/java/com/story/mipsa/attendancetracker/ExtraClassEdit.java

```diff
@@ -67,9 +61,17 @@ public void onClick(View view) {
     public void onAttach(Context context) {
         super.onAttach(context);
         try {
-            onInput1 = (extraClassEdit.onInput1) getActivity();
+            onInput1 = (ExtraClassEdit.onInput1) getActivity();
         } catch (ClassCastException e) {
 
         }
     }
+
+    private void shakeItBaby() {
+        if (Build.VERSION.SDK_INT >= 26) {
+            ((Vibrator) getContext().getSystemService(VIBRATOR_SERVICE)).vibrate(VibrationEffect.createOneShot(125, VibrationEffect.DEFAULT_AMPLITUDE));
+        } else {
+            ((Vibrator)getContext().getSystemService(VIBRATOR_SERVICE)).vibrate(125);
+        }
+    }
```

Comment:
Since this function is used in 3-4 places, should move it to a common Util class to avoid code duplication. Can ignore for now.